kiwi_post_run

```diff
@@ -98,18 +91,23 @@ echo "version $VERSION"
 echo "release $RELEASE"
 echo "source $IMAGE"
 echo "metadata $METADATA"
+echo "tagfile $TAGFILE"
 
 # Generate metada JSON file for container-feeder
 cat << EOF > $METADATA
 {
   "image": {
     "name": "$CONTAINER_RAW",
-    "tags": [ "$CONTAINER_TAG", "latest" ],
+    "tags": [ "${VERSION}-${RELEASE}", "${CONTAINER_TAG}-${RELEASE}", "$CONTAINER_TAG", "latest" ],
```

Comment:
I understand, but then it gets inconsistent because the image is generated with the tag in the kiwi spec file, so you will have haproxy:1.6.0 and haproxy:1.6.11 which are actually the same.

If you update the kiwi spec file, you may want to update the version in it, to show it is an update, but then this will not correlate with the version of the main package it contains. I think it is not good to have version and tag differently because either you use one or the other you mess up if they are different ... it is at least confusing